src/current/cockroachcloud/stream-changefeed-to-snowflake-aws.md

```diff
@@ -167,44 +153,44 @@ If your changefeed is running but data is not displaying in your S3 bucket, you
     ~~~ sql
-    > CREATE STAGE cdc_stage url='s3://changefeed-example/' credentials=(aws_key_id='<KEY>' aws_secret_key='<SECRET_KEY>') file_format = (type = json);
+    CREATE STAGE cdc_stage url='s3://changefeed-example/' credentials=(aws_key_id='<KEY>' aws_secret_key='<SECRET_KEY>') file_format = (type = json);
     ~~~
 
-    Be sure to replace the placeholders with your AWS access key ID and AWS secret access key.
+    Replace the placeholders with your AWS access key ID and AWS secret access key.
 
-1. In the Worksheet, create a Snowpipe called `cdc-pipe`, which tells Snowflake to auto-ingest data:
+1. In the worksheet, create a Snowpipe called `cdc-pipe`, which tells Snowflake to auto-ingest data:
 
     {% include_cached copy-clipboard.html %}
     ~~~ sql
-    > CREATE PIPE cdc_pipe auto_ingest = TRUE as COPY INTO order_alerts FROM @cdc_stage;
+    CREATE PIPE cdc_pipe auto_ingest = TRUE as COPY INTO order_alerts FROM @cdc_stage;
     ~~~
 
     {{site.data.alerts.callout_info}}
     Auto-ingest in Snowflake works with [AWS](https://docs.snowflake.com/user-guide/data-load-snowpipe-auto-s3.html), [Azure](https://docs.snowflake.net/manuals/user-guide/data-load-snowpipe-auto-azure.html), and [Google Cloud Storage](https://docs.snowflake.com/user-guide/data-load-snowpipe-auto-gcs.html).
     {{site.data.alerts.end}}
 
-1. In the Worksheet, view the Snowpipe:
+1. In the worksheet, view the Snowpipe:
 
     {% include_cached copy-clipboard.html %}
     ~~~ sql
-    > SHOW PIPES;
+    SHOW PIPES;
     ~~~
 
-1. Copy the **ARN** of the SQS queue for your stage (this displays in the **notification_channel** column). You will use this information to [configure the S3 bucket](#step-9-configure-the-s3-bucket).
+1. Copy the **ARN** of the SQS queue for your stage, which displays in the **notification_channel** column. You will use this information to [configure the S3 bucket](#step-9-configure-the-s3-bucket).
 
 ## Step 9. Configure the S3 bucket
 
 1. [Navigate back to your S3 bucket](https://s3.console.aws.amazon.com/).
 
-1. Configure an event notification for the S3 bucket. Use the following parameters:
-    - **Name:** Name of the event notification (e.g., Auto-ingest Snowflake).
-    - **Events:** Select the **All object create events**.
-    - **Send to:** Select **SQS Queue** from the drop-down.
-    - **SQS:** Select **Add SQS queue ARN** from the drop-down.
+1. Configure an event notification for the S3 bucket. In the **Properties** tab, click **Create event notification**. Use the following parameters:
+    - **Event Name:** Name of the event notification (e.g., Auto-ingest Snowflake).
+    - **Event types:** Select the **All object create events**.
+    - **Destination:** Select **SQS Queue**.
+    - **Specify SQS queue:** Select **Enter SQS queue ARN** from the drop-down.
     - **SQS queue ARN:** Paste the SQS queue name from the `SHOW PIPES` output (from [Step 8](#step-8-configure-snowflake)).
```

Comment:
```suggestion
    - **Specify SQS queue:** Select **Enter SQS queue ARN** from the drop-down.
    - **SQS queue ARN:** Paste the SQS queue name from the `SHOW PIPES` output (from [Step 8](#step-8-configure-snowflake)).
```

nit: nest these two points within **Destination** as they are parameters within the Destination configuration

Reply:
I don't think this quite makes a difference here. They're actually in a separate section in the UI. And as we're just directing the user through each click, it kind matters more what is happening in our instructions.

Reply:
Got it. When looking at the S3 page, I see that it is under the Destination section but the following flow already guides a user down the page and works.